ci-operator/step-registry/ipi/conf/vsphere/windows/dns/ipi-conf-vsphere-windows-dns-commands.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
This is a lot of duplication. You shouldn't need to create the `api` or `*.apps` that should have already been done.
I would rather see the addition of `api-int` since it won't be used by anything other than Windows:
`ci-operator/step-registry/ipi/conf/vsphere/dns/ipi-conf-vsphere-dns-commands.sh`